conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
 - fastavro>=0.22.0
 - notebook>=0.5.0
 - cython>=0.29,<0.30
+- fsspec>=0.3.3
```

Comment:
Current dependency list includes `s3fs`. Could we remove this from the dependencies and just keep it in the CI dependencies to run tests

Reply:
Yes

Reply:
Thanks. Just to reconfirm this would mean no user would be able to use io methods to read data without installing fsspec separately in their environment.

Reply:
Sorry, maybe I misunderstood. I thought you were referring to removing `s3fs` not `fsspec`. What are you trying to accomplish?

Reply:
Whoops. You are right. Intention was the remove s3fs. Got confused while reading my review now and replying.

Update: fsspec **will** be a dependency. S3fs will **not**

Reply:
Done